rewrite-core/src/test/java/org/openrewrite/PathUtilsTest.java

```diff
@@ -37,10 +37,14 @@ void globMatching() {
         assertThat(matchesGlob(path("a\\b\\c"), "a/b/c")).isTrue();
         assertThat(matchesGlob(path("/test"), "/test")).isTrue();
         assertThat(matchesGlob(path("test"), "/test")).isFalse();
+        assertThat(matchesGlob(path("a/test"), "/test")).isFalse();
+        assertThat(matchesGlob(path("a/b/test"), "/test")).isFalse();
         assertThat(matchesGlob(path("/test"), "test")).isFalse();
         assertThat(matchesGlob(path("test/"), "test/")).isFalse(); // Trailing slash on path is not maintained
         assertThat(matchesGlob(path("test"), "test/")).isFalse();
         assertThat(matchesGlob(path("test/"), "test")).isTrue(); // Trailing slash on path is not maintained
+        assertThat(matchesGlob(path("gradle.properties"), "gradle.properties")).isTrue();
+        assertThat(matchesGlob(path("a/gradle.properties"), "gradle.properties")).isFalse();
```

Comment:
Any reason for gradle specific stuff here? It appears to me when I see this assertion that there is specific behaviour for gradle.properties.
What do you think about this?
```suggestion
        assertThat(matchesGlob(path("filename.ext"), "filename.ext")).isTrue();
        assertThat(matchesGlob(path("a/filename.ext"), "filename.ext")).isFalse();
```